src/integrationTest/java/io/aiven/kafka/connect/jdbc/oracle/VerifyDeleteIT.java

```diff
@@ -0,0 +1,276 @@
+        return new ProducerRecord<>(TEST_TOPIC_NAME, partition, String.valueOf(id), record);
+    }
+
+    private ProducerRecord<String, GenericRecord> createTombstoneRecord(
+            final int id, final int partition) {
+        return new ProducerRecord<>(TEST_TOPIC_NAME, partition, String.valueOf(id), null);
+    }
+
+    private void sendTestData(final int numberOfRecords) throws InterruptedException, ExecutionException {
+        final List<Future<RecordMetadata>> sendFutures = new ArrayList<>();
+        for (int i = 0; i < numberOfRecords; i++) {
+            for (int partition = 0; partition < TEST_TOPIC_PARTITIONS; partition++) {
+                final String recordName = "user-" + i;
+                final String recordValue = "value-" + i;
+                final ProducerRecord<String, GenericRecord> msg = createRecord(i, partition, recordName, recordValue);
+                sendFutures.add(producer.send(msg));
+            }
+        }
+        producer.flush();
+        for (final Future<RecordMetadata> sendFuture : sendFutures) {
+            sendFuture.get();
+        }
+    }
+
+    private void sendTestDataWithTombstone(final int numberOfRecords) throws InterruptedException, ExecutionException {
+        final List<Future<RecordMetadata>> sendFutures = new ArrayList<>();
+        for (int i = 0; i < numberOfRecords; i++) {
+            for (int partition = 0; partition < TEST_TOPIC_PARTITIONS; partition++) {
+                final ProducerRecord<String, GenericRecord> record = createTombstoneRecord(i, partition);
+                sendFutures.add(producer.send(record));
+            }
+        }
+        producer.flush();
+        for (final Future<RecordMetadata> sendFuture : sendFutures) {
+            sendFuture.get();
+        }
+    }
+
+    private void sendMixedTestDataWithTombstone(final int numberOfRecords, final int numberOfTombstoneRecords)
+            throws InterruptedException, ExecutionException {
+        final List<Future<RecordMetadata>> sendFutures = new ArrayList<>();
+        for (int i = 0; i < numberOfRecords; i++) {
+            for (int partition = 0; partition < TEST_TOPIC_PARTITIONS; partition++) {
+                final String recordName = "user-" + i;
+                final String recordValue = "value-" + i;
+                final ProducerRecord<String, GenericRecord> msg = createRecord(
+                        i, partition, recordName, recordValue);
+                sendFutures.add(producer.send(msg));
+                if (i < numberOfTombstoneRecords) {
+                    final ProducerRecord<String, GenericRecord> record = createTombstoneRecord(i, partition);
+                    sendFutures.add(producer.send(record));
+                }
+            }
+        }
+
+        producer.flush();
+        for (final Future<RecordMetadata> sendFuture : sendFutures) {
+            sendFuture.get();
+        }
+    }
+
+    @BeforeEach
+    public void afterEach() throws SQLException {
```

Comment:
We should remove this, and revert the change to make the `@Contaner`-annotated fields `static`; otherwise, the container is reused across test cases ([ref](https://java.testcontainers.org/test_framework_integration/junit_5/)).

I understand that it may be more efficient and faster to share containers across test methods, but it's not a requirement for this PR and right now some of our integration tests are failing if more than one is run in the same build.